Cargo.toml

```diff
@@ -117,3 +117,6 @@ lto = true
 codegen-units = 1
 opt-level = 3
 strip = "symbols"
+
+[workspace.lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(nightly)'] }
```

Comment:
we should probably plan on moving all lints here from the various `lib.rs` and binary definitions, right?

Reply:
For things which are useful across the workspace, that might make sense, yes.